tests/test_managers.py

```diff
@@ -0,0 +1,43 @@
+from pytest import fixture
+from sqlalchemy import select
+
+from src.managers import DepartmentManager
+from src.models import Department, SessionLocal
+
+
+class TestCreateDepartment:
+    @fixture(scope="class", autouse=True)
+    def create(self, department_data):
+        return DepartmentManager.create(department_data)
+
+    def test_create(self, department_data):
+        query = select(1).select_from(Department).where(
+            Department.title == department_data["title"],
+            Department.country_name == department_data["country_name"],
+        ).exists().select()
+        with SessionLocal() as session:
+            assert session.execute(query).scalar()
+
+
+class TestGetDepartment:
+    @fixture(scope="class", autouse=True)
+    def get_all(self):
+        return DepartmentManager.get_all()
+    
+    def test_get_all(self):
+        query = select(1).select_from(Department).exists().select()
+        with SessionLocal() as session:
+            assert session.execute(query).scalar()
```

Comment:
In TestCreate I checked if new department is created or not, so `exists` was appropriate check. What do you check here? I mean, what should be checked? I would check amount of returned records and records' data (returned from  the tested method, not from the database)